irispy/new_sji.py

```diff
@@ -288,6 +288,77 @@ def world_axis_physical_types(self):
 
     def plot(self, axes=None, plot_axis_indices=None, axes_coordinates=None,
              axes_units=None, data_unit=None, **kwargs):
+        """
+        Visualizes data in the IRISMapCubeSequence with the sequence axis folded
+        into the common axis.
+
+        Based on the cube-like dimensionality of the sequence and value of plot_axis_indices
+        kwarg, a Line/Image Plot/Animation is produced.
+
+         Parameters
+        ----------
+        axes: `astropy.visualization.wcsaxes.core.WCSAxes` or ??? or None.
+            The axes to plot onto. If None the current axes will be used.
+
+        plot_axis_indices: `int` or iterable of one or two `int`.
+            If two axis indices are given, the sequence is visualized as an image or
+            2D animation, assuming the sequence has at least 2 cube-like dimensions.
+            The cube-like dimension indicated by the 0th index is displayed on the
+            x-axis while the cube-like dimension indicated by the 1st index is
```

Comment:
The x-axis index is the first element of ```plot_axis_indices```.  So ```plot_axis_indices=[0, 1]``` will have the 0th dimensions of the cube of the x-axis of the plot.  But ```plot_axis_indices=[1, 0]``` should have the axes inverted.

Reply:
I think the text is correct because the 0th and 1st index are here the index of `plot_axis_indices`.

Reply:
Ah OK.  You're right.  But it is confusing.  Could you replace ```The cube-like dimension indicated by the 0th index is displayed...``` with something like ```The cube-like dimension indicated by the 0th index of plot_axis_indices is displayed...```?

Reply:
Or better yet, ```0th element of plot_axis_indices``` instead of ```0th index of plot_axis_indices```.